rpm/ndb/ndb.go

```diff
@@ -14,8 +14,9 @@ var le = binary.LittleEndian
 
 // Used throughout the various DBs.
 const (
-	slotSize  = 4 * 4
-	slotStart = 2
+	slotSize           = 4 * 4
+	slotStart          = 2
+	slotEntriesPerPage = 4096 / 16
```

Comment:
Where does this come from?

Reply:
https://github.com/rpm-software-management/rpm/blob/master/lib/backend/ndb/rpmpkg.c#L175
https://github.com/rpm-software-management/rpm/blob/master/lib/backend/ndb/rpmpkg.c#L177